README.md

```diff
@@ -1,52 +1,167 @@
 # Ajax and Callbacks
-
 ## Objectives
-+ Explain what an API is and why it's used
-+ Explain what Ajax is and why it's used
-+ Make a `get` request using Ajax to append text to a page
-+ Explain what a callback is and why Ajax runs asynchronously
+- Explain what an API is and why it's used
+- Explain what Ajax is and why it's used
+- Make a get request using Ajax to append text to a page
+- Explain what a callback is and why Ajax runs asynchronously
```

Comment:
These objectives have lots of explanation and not a lot of action — maybe we can rewrite the last objective so that it focuses on handling asynchronicity with a callback (or we could include a last objective for doing so), rather than just explaining it?